### DISCUSSION TITLE
How to draw Elbow Arrow Connector In Fabric Canvas

### DISCUSSION THREAD

**Discussion by Shilpa-K07** [fabricjs/fabric.js#7350] [Q&A]
I want to connect two rectangle shapes using arrow connector. I'm able to draw simple straight arrow between the shapes.
But I want something like Elbow connector 

![image](https://user-images.githubusercontent.com/70196456/131492260-b235e074-1339-43c8-a5ea-813dede77dc9.png)

Please help me with this.

**Comment by ShaMan123** [comment]
I have implemented this behavior.
You should perform some kind of calculation to create that path points in accordance to what you want.
Then add fabric.Path to canvas or better off, create a custom object that extends fabric.Path
